Suit icon rework no 1

Pebblesthe2nd opened the pull request:
Made/reworked icons for a bunch of suits.
Adjusted a couple onmob sprites to match.
Goal: Make clothes look better, get a new baseline going beyond just copying the onmob sprite for icons, both for looks and scale of sprites.

Preview https://imgur.com/a/kSeVRew
After some edits https://imgur.com/pqhbhdS

Pebblesthe2nd committed:
Suit icon rework

Made icons for a bunch of suits.
Adjusted a couple onmob sprites to match.
Goal: Make clothes look better, get a new baseline going beyond just copying the onmob sprite for icons, both for looks and scale of sprites.

Preview https://imgur.com/a/kSeVRew

Pebblesthe2nd committed:
Couple minor tweaks

some jackets and labcoats got a pixel or two shift

SierraKomodo reviewed (changes requested):
The CSO jacket sprites don't quite match up - On mob has the black accents, item icon is missing the black accents now.

![chrome_KoymqW51Tt](https://user-images.githubusercontent.com/11140088/104964328-0ee5c000-5991-11eb-9583-554ecc3c0501.png)

![dreamseeker_fKxNG9JcmJ](https://user-images.githubusercontent.com/11140088/104964900-4acd5500-5992-11eb-803b-439cd83ac62a.png)

![dreamseeker_yaHdaIRd13](https://user-images.githubusercontent.com/11140088/104964903-4bfe8200-5992-11eb-9af8-ddb0603af208.png)

Pebblesthe2nd committed:
Update

Tweaks to increase difference between bulkier and slimmer clothing.
Align CSO onmob/icon.
Adding some track jackets.

Pebblesthe2nd committed:
Update onmob_suit.dmi

Pebblesthe2nd committed:
Update onmob_suit.dmi

Why is this not working =/

Pebblesthe2nd commented:
Pulled the latest suits onmob file to avoid the conflict, now it fails check in one second. Weird. "Unable to render pull requests in an unmergeable state" tells me nothing. Any way to sort this without closing the PR and opening a new one?

SierraKomodo commented:
> Pulled the latest suits onmob file to avoid the conflict, now it fails check in one second. Weird. "Unable to render pull requests in an unmergeable state" tells me nothing. Any way to sort this without closing the PR and opening a new one?

Github still shows a merge conflict. Try rebasing off dev instead of merging it into your PR to resolve the conflict.

Pebblesthe2nd commented:
Can't figure out how to do that, fixing it the only way I know how, deleting and resetting.
